.github/actions/tfplus-python-test/action.yml

```diff
@@ -1,11 +1,11 @@
 ---
-name: tfplus-python-test
+name: tfplus-test
```

Comment:
also rename the outside folder to "tfplus-test", if you removed the action name.